Cargo.toml

```diff
@@ -35,10 +35,12 @@ tokio = { version = "1.29.1", optional = true, features = ["fs", "macros"] }
 ureq = { version = "2.8.0", optional = true, features = [
   "json",
   "socks-proxy",
+  "native-tls"
 ] }
+native-tls = { version = "0.2", optional = true }
 
 [features]
-default = ["default-tls", "tokio", "ureq"]
+default = ["default-tls", "tokio", "ureq", "native-tls"]
```

Comment:
I'm confused by this ?

 Why adding another tls feature ?
 
 `default-tls` is defined in reqwest. For now it means native-tls.
 The idea behind it is that the default could change (breaking semver) at some point. I think we should keep that semantic.
 
 So `native-tls` could exist, but it should be exclusive with `default-tls` and would mean exactly : get me native-tls.